marklogic-data-hub/src/test/java/com/marklogic/hub/AbstractHubTest.java

```diff
@@ -1,16 +1,20 @@
 package com.marklogic.hub;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 
 /**
- * Adding this so that a class can subclass HubTestBase without having to define the same two Spring annotations
+ * Adding this so that a class can subclass HubTestBase without hing to define the same two Spring annotations
```

Comment:
A small typo here. Other than that everything looks good.

Reply:
Thanks @srinathgit  - I'll have another PR for 4440 with the real impl in it; I'll fix the typo in that one.